tests/trainer/data_flow/test_train_loop_flow_dict.py

```diff
@@ -24,7 +22,6 @@
 from tests.helpers.deterministic_model import DeterministicModel
 
 
-@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
```

Comment:
maybe leave this `PL_DEV_DEBUG` for a separate PR so this is focus on one thing only